sw/simulator/nps/nps_fdm_jsbsim.cpp

```diff
@@ -508,8 +508,18 @@ static void init_jsbsim(double dt)
   string rootdir;
   string jsbsim_ic_name;
 
-  sprintf(buf, "%s/conf/simulator/jsbsim/", getenv("PAPARAZZI_HOME"));
-  rootdir = string(buf);
+  char* pprz_home = getenv("PAPARAZZI_HOME");
+
+  if (strlen(pprz_home) < sizeof(buf)) {
+    sprintf(buf, "%s/conf/simulator/jsbsim/", pprz_home);
```

Comment:
I am curious to see what will coverity think. Imho should be safe as we already checked upper bounds of the size.